analyzer-models/src/test/java/ffb/test/utilities/GenericTests.java

```diff
@@ -0,0 +1,52 @@
+package ffb.test.utilities;
+
+import ffb.analyzer.models.espn.TransactionCounter;
+import org.junit.Assert;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class GenericTests {
+    public static void validateGetMethodsReturnNonNullValue(List<?> testObjects) throws InvocationTargetException, IllegalAccessException {
+        List<Method> getterMethods = new ArrayList<Method>();
+
+        for (Method method : testObjects.get(0).getClass().getMethods()) {
+            if (isGetterMethod(method)) {
+                getterMethods.add(method);
+            }
+        }
+
+        for (var i = 0; i < testObjects.size(); i++) {
```

Comment:
you can also do:

```java
for (testObj : testObjects) {
    for (method: methods) {
    .....
    }
}
```

to remove all those gets.

Reply:
It should also be `int i` and `int j` for your indices.

Reply:
I like that shorthand a lot better. Updated. And noted for the indices.